src/resources/js/views/datepicker.js

```diff
@@ -371,6 +379,117 @@ tribe.events.views.datepicker = {};
 		};
 	};
 
+	/**
+	 * Set today to date object representing today
+	 *
+	 * @since TBD
+	 *
+	 * @param {string} today string representation of today's date according to website time
+	 *
+	 * @return {void}
+	 */
+	obj.setToday = function( today ) {
+		var date = today.split( ' ' )[0];
```

Comment:
I would like for us to check if a ` ` is available, if not just consider a date without time.